### PR TITLE
Add support for negative values for reg limits

### PR DESCRIPTION
**Pull request by odinuge**

### CODE REVIEW

**Review by SmithPeder — APPROVED** [review]

File: app/routes/events/components/EventAdministrate/Attendees.js [hunk 1]
```diff
@@ -107,7 +107,10 @@ export default class Attendees extends Component<Props, State> {
     if (error) {
       return <div>{error.message}</div>;
     }
-    const showUnregister = moment().isBefore(event.startTime);
+    const showUnregister =
+      moment().isBefore(event.startTime) ||
+      moment().isBefore(event.unregistrationCloseTime) ||
+      moment().isBefore(event.registrationCloseTime);
```
Comment (LudvigHz) [inline app/routes/events/components/EventAdministrate/Attendees.js:113]: What's the reasoning behind this? Shouldn't admins be able to admin unregister after registration closes?

Anyways, this doesn't really make sense. If `unregistrationCloseTime` is before `registrationCloseTime`, the button will show, but won't work.

**Review by SmithPeder — APPROVED** [review]

**Comment by SmithPeder** [comment]
Ups wrong PR

File: app/routes/events/components/EventAdministrate/Attendees.js (hunk 1 above)
Comment (odinuge) [reply app/routes/events/components/EventAdministrate/Attendees.js:113]: What is your preferred way then? Doesn't really matter to me.

Comment (odinuge) [reply app/routes/events/components/EventAdministrate/Attendees.js:113]: This logic is dead (together with the GDPR logic here) anyway I guess, so it doesn't really matter

Comment (odinuge) [reply app/routes/events/components/EventAdministrate/Attendees.js:113]: > Shouldn't admins be able to admin unregister after registration closes?

They can with this equally much as before

Comment (odinuge) [reply app/routes/events/components/EventAdministrate/Attendees.js:113]: Backend don't care tho., so it doesn't matter

**Review by LudvigHz — APPROVED** [review]

**Pull request merged**